contributing/versions.md

```diff
@@ -13,23 +13,27 @@ MDC's version numbers strictly follow [semantic versioning](http://semver.org/):
 
 Major releases can contain breaking changes to clients, while minor and patch releases cannot.
 
-## What are breaking changes?
+## How do we determine which numbers need to change?
 
-- API deletions or modifications.
+We follow the guidelines defined by the [semver v2.0.0](https://semver.org/spec/v2.0.0.html#semantic-versioning-specification-semver) specification.
+
+## Roughly-speaking, what are breaking changes?
+
+- Build-breaking modifications for either Objective-C or Swift code.
 - Visible changes to user interface components.
 
-## What are non-breaking changes?
+## Roughly-speaking, what are non-breaking changes?
 
 - API additions.
 - Behavioral changes.
 
-## How can a release be a patch release?
+## Roughly-speaking, how can a release be a patch release?
 
 Either:
 
 - There are *no changes to component sources* (changes to documentation—README.md or
 header docs—do not count), or
-- Component changes *only include bug fixes with no apparent behavioral changes*.
+- Component changes *only include bug fixes*.
```

Comment:
This seems to be at odds with the previous 2 definitions. If I fix a bug and it has no (1) visible change and has no (2) behavior change, then what's left?

Reply:
After removing `no apparent behavioral changes` and changing to `bug fixes`, I feel it is clear and also aligned to the guideline in https://semver.org/spec/v2.0.0.html. Bug fixes can have visible change or/and behavior change, but the change/bug fix should be correcting the old wrong behavior instead of adding incremental functionalities/behaviors.